src/core/src/op/constant.cpp

```diff
@@ -552,6 +552,7 @@ bool ov::op::v0::Constant::visit_attributes(AttributeVisitor& visitor) {
 bool ov::op::v0::Constant::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const {
     OV_OP_SCOPE(v0_Constant_evaluate);
     auto output = outputs[0];
+    output->set_shape(get_shape());
```

Comment:
For operations other than `Constant` such `get_shape()` on node is not safe to call in evaluate, 
but it seems to be safe for `Constant` because there is guarantee it has static shape.

Reply:
Generally in evalute methods the inputs should have only static shapes only (most evaluates assume static shape and gets shape from inputs), Now is possible to have dynamic because of conversion from ov::Tensor (Shape{0}) will produce dynamic Host tensor. 
But true, consent has always static shape.